app/controllers/WKRemoteDataStoreController.scala

```diff
@@ -145,7 +145,7 @@ class WKRemoteDataStoreController @Inject()(
     }
   }
 
-  def deleteErroneous(name: String, key: String): Action[JsValue] = Action.async(parse.json) { implicit request =>
+  def deleteDatasource(name: String, key: String): Action[JsValue] = Action.async(parse.json) { implicit request =>
```

Comment:
```suggestion
  def deleteDataset(name: String, key: String): Action[JsValue] = Action.async(parse.json) { implicit request =>
```

I’d go for dataset here as this is on the wk side and specifically operates on the datasets table